arm-notificationhubs/2016-03-01/swagger/notificationhubs.json

```diff
@@ -129,6 +129,52 @@
           }
         }
       },
+	  "patch": {
+		"tags": [
+          "Namespaces"
+        ],
+		"operationId": "Namespaces_Patch",
```

Comment:
That is the recommended pattern. That makes things clear to the customer that it is a PATCH operation.